src/scythe-bidder/game.ts

```diff
@@ -9,9 +10,14 @@ import {
   Mat,
 } from "./types";
 
-const matToIdx: { [key: string]: number } = {};
-mats.forEach((mat, idx) => {
-  matToIdx[mat] = idx;
+const matToIdxBase: { [key: string]: number } = {};
+mats_base.forEach((mat, idx) => {
+  matToIdxBase[mat] = idx;
+});
```

Comment:
Is this being used? I think you could get away with just `matToIdx` because it's just being used for determining the first player, and the relative order of the base mats is still the same in IFA?

Reply:
Again, to be cleaned up once setupData is used.